docs/docs/code/python/faqs/README.md

```diff
@@ -0,0 +1,25 @@
+# Frequently Asked Questions about Python
+
+## What's the difference between `def handler(pd)` and the `pipedream` package for Python code steps?
+
+The `handler` method is a newer implementation of Python code steps. It includes the ability to stop the workflow early, integrate a Data Store and/or Connected Accounts into your Python code step.
+
+However, at this time there are issues with our Python interpreter that is causing an `ECONNRESET` error.
+
+At this time, we recommend using the `pipedream` package to import and export data from Python code steps to other steps in your workflow.
```

Comment:
I'd clarify that users should use the `pipedream` package when they _only_ need to import / export data and don't need access to the other features. I'd also link to the relevant code example using the `pipedream` package so people see how that works.